test/IISIntegration.FunctionalTests/Inprocess/StartupTests.cs

```diff
@@ -183,6 +183,23 @@ public StartupTests(ITestOutputHelper output) : base(output)
             Assert.Contains(testSink.Writes, context => context.State.ToString().Contains("Application is running inside IIS process but is not configured to use IIS server"));
         }
 
+        [Fact]
+        public async Task LoadsAspNetCoreInProcessDll()
+        {
+            using (StartLog(out var loggerFactory))
```

Comment:
What's the point of this test? Aren't we PInvoking into `aspnetcorev2_inprocess` for every operation anyway ?

Reply:
It's just to make sure the name of the dll doesn't change. I'm working on native tests now.

Reply:
Also I'll rename the test.

Reply:
But why do we care? When do you expect this test to fail but not the others?